app/views/sal3_batch_requests_batches/index.html.erb

```diff
@@ -2,23 +2,29 @@
 <table class="table table-striped">
   <thead>
     <tr>
-      <th class='col-xs-1'>Batch</th>
-      <th class='col-xs-2'>Batch name</th>
-      <th class='col-xs-3'>User name</th>
-      <th class='col-xs-4'>User email</th>
-      <th class='col-xs-5'>Comments</th>
+      <th class='col-xs-1'></th>
+      <th class='col-xs-2'>Date needed</th>
+      <th class='col-xs-3'>Name or short description</th>
+      <th class='col-xs-4'>Stop code</th>
+      <th class='col-xs-5'>No. of items requested</th>
+      <th class='col-xs-6'>Status</th>
+      <th class='col-xs-7'>Multiple requests in batch?</th>
+      <th class='col-xs-8'>Date requested</th>
+      <th class='col-xs-9'></th>
     </tr>
   </thead>
   <tbody>
     <% @sal3_batch_requests_batches.each do |batch| %>
       <tr class='<%= cycle("odd", "even") %>'>
-        <td class='col-xs-1'><%= link_to batch.batch_id, batch %></td>
-        <td class='col-xs-2'><%= batch.batch_name %></td>
-        <td class='col-xs-3'><%= batch.user_name %></td>
-        <td class='col-xs-4'><%= batch.user_email %></td>
-        <td class='col-xs-5'><%= batch.comments %></td>
-        <td class='col-xs-6'><%= link_to 'Edit', edit_sal3_batch_requests_batch_path(batch), class: 'btn btn-primary' %></td>
-        <td class='col-xs-6'><%= link_to 'Show', sal3_batch_requests_batch_path(batch), class: 'btn btn-primary' %></td>
+        <td class='col-xs-1'><%= link_to 'Batch details', sal3_batch_requests_batch_path(batch), class: 'btn btn-default' %></td>
+        <td class='col-xs-2'><%= batch.batch_needbydate.strftime('%x') %></td>
+        <td class='col-xs-3'><%= batch.comments %></td>
```

Comment:
I think this should be batch.batch_name.  There is a separate column in the table for batch.comments, but that's not this.